libraries/hermes/src/classes/ProxyRoute.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from 'express';
+import { createProxyMiddleware } from 'http-proxy-middleware';
+import { ProxyRouteOptions } from '../interfaces';
+import { ConduitProxy, Indexable } from '@conduitplatform/grpc-sdk';
+
+export class ProxyRoute {
+  private readonly _input: ProxyRouteOptions;
+
+  constructor(input: ProxyRouteOptions) {
+    this._input = input;
+  }
+
+  get input(): ProxyRouteOptions {
+    return this._input;
+  }
+
+  executeRequest(req: Request, res: Response, next: NextFunction) {
+    createProxyMiddleware(this._input)(req, res, next);
```

Comment:
the proxy middleware is created without the devs options, or any options as a matter of fact